pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "mex-common"
-version = "0.18.1"
+version = "0.19.0"
```

```suggestion
version = "0.19.dev0"
```

if we are doing releases [by the book](https://packaging.python.org/en/latest/specifications/version-specifiers/#developmental-releases) now, it'd have to be something like this. but let's discuss in the office.

---

As discussed, let's stick to the previous release process for this PR and come up with a strategy for future releases including the format of the changelog together on Tuesday?